missing_types.h

```diff
@@ -0,0 +1,5 @@
+#include <stdint.h>
+
+typedef int16_t sint16;
+typedef uint16_t uint16;
+typedef uint8_t uint8;
```

Comment:
is that necessary after last merge?

Reply:
haha, I see you had the issue too.
Let me check !

Reply:
It was indeed not necessary anymore :)